packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/test/vtmnRadioButton.spec.js

```diff
@@ -0,0 +1,41 @@
+import '@testing-library/jest-dom';
+import { render, fireEvent } from '@testing-library/svelte';
+import VtmnRadioButton from '../VtmnRadioButton.svelte';
+
+const id = 'id';
+const name = 'name';
+const label = 'label';
+
+const params = { id, name, label };
+
+describe('VtmnRadioButton', () => {
+  test('Radio Button Should be visible', () => {
+    const { getByLabelText, getByText } = render(VtmnRadioButton, params);
+    expect(getByLabelText(label)).toBeInTheDocument();
+    expect(getByText(label)).toBeInTheDocument();
+  });
+
+  test('Radio Button Should be selectable', () => {
+    const { getByLabelText } = render(VtmnRadioButton, params);
+    const radioButton = getByLabelText(label);
+    expect(radioButton).not.toBeChecked();
+    fireEvent.click(radioButton);
```

Comment:
`await fireEvent.click(...)` and the function should be `async`

Reply:
I'm not sure, on [doc](https://testing-library.com/docs/dom-testing-library/api-events/) they don't use async func for this.

Reply:
This one is the example for svelte ->. https://testing-library.com/docs/svelte-testing-library/example